Sources/SwiftDocC/Model/DocumentationNode.swift

```diff
@@ -148,7 +148,8 @@ public struct DocumentationNode {
     /// > Warning: Using this initializer will not report any diagnostics raised during the initialization of `DocumentationNode`.
     @available(*, deprecated, message: "Use init(reference:symbol:platformName:moduleName:article:engine:) instead")
     public init(reference: ResolvedTopicReference, symbol: SymbolGraph.Symbol, platformName: String?, moduleName: String, article: Article?, problems: inout [Problem]) {
```

Comment:
It would be good if to remove this initializer. It's been deprecated for a while now.